shared-components/Icon/Comment.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { memo } from "react";
+import { SVGIconProps } from "./";
+
+const CommentIcon = ({ height = "24px", onClick, style, className }: SVGIconProps) => (
+  <svg
+    width={height}
+    height="24"
+    viewBox="0 0 24 24"
+    fill="none"
+    style={style}
+    onClick={
+      onClick
+        ? onClick
+        : () => {
+            console.log("click");
+          }
+    }
+    className={className}
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      opacity="0.7"
+      d="M18.1498 5H5.85015C5.3792 5 5 5.3792 5 5.85015V13.6911V15.0612V18.2844L7.30581 15.9174H18.1498C18.6208 15.9174 19 15.5382 19 15.0673V5.85015C19 5.3792 18.6208 5 18.1498 5Z"
+      stroke="#101020"
```

Comment:
여기서 이게 fill의 역할을 하니 fill의 기본값을 #101020으로 하고 넣어주세요
동작에 따라 색이 변할 수도 있어요

```suggestion
      stroke={fill}
```